practice-app/schemas.py

```diff
@@ -0,0 +1,28 @@
+from typing import List, Optional
+from pydantic import BaseModel
+
+
+class CreateBook(BaseModel):
```

Comment:
It looks like the CreateBook class is not used and it is very similar to the Book class, can you explain why do we need it?

Reply:
I will need this in post method. Users will enter these fields to store a book in database.

Reply:
I see, it makes sense.